README.md

```diff
@@ -60,6 +60,7 @@ Reproduce a ci failure
 ----------------------
 
 ```
-SELENIUM_IMAGE_SUFFIX=-debug docker-compose -f docker-compose.yml -f docker-compose.ci.yml up --build -V
+docker-compose down -v
```

Comment:
Not needed? (Problem's only in `docker-compose.override.yml`?)

Reply:
There is [a volume in the ci configuration](https://github.com/elifesciences/journal/blob/develop/docker-compose.ci.yml#L32) so this should really be executed with the `ci` configuration if we don't want to carry over code from a previous execution.